backend/db/refresh.py

```diff
@@ -46,7 +47,8 @@ def refresh_db(
     local = use_local_db
     print('INFO: Starting database refresh.', flush=True)  # flush: for gh action
     t0, t0_str = datetime.now(), current_datetime()
-    if check_db_status_var('refresh_status') == 'active':
+
+    if is_refresh_active():
```

Comment:
This now replaces the check `if check_db_status_var('refresh_status') == 'active':`